src/backend/LDW.Application/Features/DormitoryFeatures/Commands/UploadDormitoryPictureCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿using LDW.Application.Interfaces;
+using LDW.Application.Models.Dormitory;
+using LDW.Domain.Entities;
+using MediatR;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace LDW.Application.Features.DormitoryFeatures.Commands
+{
+
+	public class UploadDormitoryPictureCommand : IRequest<DormitoryPictureModel>
+	{
+		public int DormitoryId { get; set; }
+		public bool IsMain { get; set; }
+		public byte[] Image { get; set; }
```

Comment:
Do we save images in db? We have already configured azure blob container (e.g for saving user profile pics). Adding picture to some dormitory, we could just save URL of the image to the db.

Reply:
oh, did't know, Will change

Reply:
I guess it's ok for now)

Reply:
i already changed :)